app/views/layouts/landing_page_layout.html.erb

```diff
@@ -62,7 +62,59 @@
     <%= render 'layouts/phase_banner' %>
   </div>
 
+  <main class="govuk-main-wrapper" id="main-content">
+    <div class="govuk-width-container govuk-!-margin-bottom-7">
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds landing-page_header-text-container govuk-!-margin-bottom-0 govuk-!-padding-bottom-0">
+          <h1 class="govuk-heading-xl govuk-!-margin-bottom-4">
+            Help for early years providers
+          </h1>
+          <p class="govuk-body-l">Information, guidance, practical support and training for delivering the early years
+            foundation stage (EYFS).</p>
+        </div>
+      </div>
+    </div>
+
+    <!--Cards section-->
+    <div class="landing-page__section--grey">
```

Comment:
I think the merge might have gone wrong.  We moved the cards section into the content controller's index view.  So when I merged main into your branch and look at this page, the cards are duplicated.